src/content/docs/en/guides/prefetch.mdx

```diff
@@ -114,6 +115,35 @@ To ignore slow connection detection, you can use the `ignoreSlowConnection` opti
 prefetch('/about', { ignoreSlowConnection: true });
 ```
 
+### `eagerness`
+
+<p><Since v="5.6.0" /></p>
```

Comment:
```suggestion
<p>
**Type:** `'immediate' | 'eager' | 'moderate' | 'conservative'`<br />
**Default:** `'immediate'`<br />
<Since v="5.6.0" />
</p>
```